apps/full-site-editing/full-site-editing-plugin/full-site-editing-plugin.php

```diff
@@ -52,6 +50,77 @@ function load_full_site_editing() {
 }
 add_action( 'plugins_loaded', __NAMESPACE__ . '\load_full_site_editing' );
 
+/**
+ * Whether or not FSE is active.
+ * If false, FSE functionality can be disabled.
+ *
+ * @returns bool True if FSE is active, false otherwise.
+ */
+function is_full_site_editing_active() {
+	return is_site_eligible_for_full_site_editing() && is_theme_supported();
+}
+
+/**
+ * Returns normalized theme slug for the current theme.
+ *
+ * Normalize WP.com theme slugs that differ from those that we'll get on self hosted sites.
+ * For example, we will get 'modern-business-wpcom' when retrieving theme slug on self hosted sites,
+ * but due to WP.com setup, on Simple sites we'll get 'pub/modern-business' for the theme.
+ *
+ * @return string Normalized theme slug.
+ */
+function get_theme_slug() {
```

Comment:
Just a note that we already have similar method declared in https://github.com/Automattic/wp-calypso/blame/17e28769440e74dcdcb3fea9d3be6e6e6d2ea509/apps/full-site-editing/full-site-editing-plugin/full-site-editing/templates/class-wp-template.php#L60 and https://github.com/Automattic/wp-calypso/blame/58f4fdf7deb635453b18f0796cd463fed54e6ecf/apps/full-site-editing/full-site-editing-plugin/full-site-editing/class-full-site-editing.php#L129.

Might be good to circle back and refactor this to avoid duplication.

Reply:
💯for sure! I originally copied it here so that we didn't have to create a class instance to access it :D